.azure-pipelines/cd-build-deploy-dotnet.yml

```diff
@@ -0,0 +1,208 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Build, sign, package and deploy .NET packages for Microsoft.Agents.M365Copilot.Core and Microsoft.Agents.M365Copilot.Beta
+
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - 'dotnet/src/Microsoft.Agents.M365Copilot.Core/**'
+      - 'dotnet/src/Microsoft.Agents.M365Copilot.Beta/Generated/**'
+      - 'dotnet/src/Microsoft.Agents.M365Copilot.Beta/AgentsM365CopilotBetaServiceClient.cs'
+  tags:
+    include: # v1.0.0-preview.2-Microsoft.Agents.M365Copilot.Beta 
+      - '*Microsoft.Agents.M365Copilot.Beta'
```

Comment:
You'll probably get the pipeline run twice on every release because both tags are included, we probably only want one to avoid failures of another systematically or race conditions oddities